packages/evolution-common/src/services/audits/types.ts

```diff
@@ -50,3 +50,15 @@ export type SurveyObjectsWithAudits = {
     home?: any; // serialized, typed in survey
     audits: AuditForObject[];
 };
+
+
+export type AuditsByLevelAndObjectType = {
+    [level: string]: {
+        [object_type: string]: {
+            key: string;
+            cnt: string;
+            object_type: string;
```

Comment:
Where are the types I should use?

Reply:
C'est que le type que tu es en train de définir, ben il a une clé au premier niveua, le level, et une clé 2e niveau, le object_type, donc la valeur n'a pas besoin d'avoir object_type et level. Sinon, tu pourrais [accidentellement peut-être, mais quand même...] avoir

```
{
    error: {
         person: [{
              key: 'qqc',
              cnt: 4,
              object_type: 'household',
              level: 'warning'
         }]
    }
}
```
Il faudrait interpréter ça comment?

Reply:
OK, I think I see, but we received them like this, otherwise, we need to add delete expressions isn't?

Reply:
And it could be useful if we move the objects later on, to be able to access their object type and level no?

Reply:
> And it could be useful if we move the objects later on, to be able to access their object type and level no?

"if"... au pire, il sera toujours possible d'ajouter les level et object_type quand on en aura besoin. J'imagine que c'est pas evil no no non plus, mais bon, ça ouvre la porte à des incohérence de données.

Reply:
Removed object_type and level from the object type